GPS.Service/Conrollers/ManageOutagesController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GPS.Service.Conrollers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ManageOutagesController : ControllerBase
+    {
+        public static IWebHostEnvironment _environment;
+
+        public ManageOutagesController(IWebHostEnvironment environment)
+        {
+            _environment = environment;
+        }
+        
+        public class FileUpload
+        {
+            public IFormFile File { get; set; }
+        }
+        
+        [HttpPost]
+        public async Task<string> Post([FromForm] FileUpload file)
+        {
+            try
+            {
+                if (file.File.Length > 0) // Make sure there's actually a file
+                {
+                    using FileStream fileStream = System.IO.File.Create(_environment.WebRootPath + "\\SOF\\" + file.File.FileName);
```

Comment:
Should create an interface to access the outage repository (IOutageRepository or something like that).  Later if we change to some other db, then none of the code will need to be modified outside of creating a new implementation of IOutageRepository.  (should have Import, Delete methods on it)

Reply:
Brayden has more notes on this

Reply:
In data service, each controller is independent and have access to same repo object